### DISCUSSION TITLE
Unable to connect to mullvad via ovpn connection.

### DISCUSSION THREAD

**Discussion by nosirrahdrof** [qdm12/gluetun#1568] [Q&A]
I assume this has some relation to this information - https://twitter.com/mullvadnet/status/1656621481386819585

Is there a manual change I can make to connect?

**Comment by jayhawks55** [comment]
> I assume this has some relation to this information - https://twitter.com/mullvadnet/status/1656621481386819585
> 
> Is there a manual change I can make to connect?

I would also appreciate to get any tips on how this can be solved.

**Comment by RogueGhost93** [comment]
This is very bad from mullvad. No communication whatsoever as to what exactly is going on, all of my wg and openvpn connections are down on different clients such as OPNsense and gluetun, cant seem to fidn what the issue is and this is becoming extremely frustrating. I appreciate any help, ideas or possible solution. Thanks!

**Comment by RogueGhost93** [comment]
I dont know why but adding this environmet variable fixed the issue for me
      - OPENVPN_CIPHERS=AES-256-GCM

If anyone figures out why or if this is even a proper fix please let us know.

**Reply by BeutelHeu** [reply]
@RogueGhost93: 

thank you! That did the job!

**Reply by Controversial-Panda** [reply]
Got it working:
Name: OPENVPN_CIPHERS
Key: OPENVPN_CIPHERS
Value: AES-256-GCM

**Reply by MurkyPuma** [reply]
Thank you, worked for me as well!!

**Comment by rymancl** [comment]
This was addressed in the v3.34.0 release.
https://github.com/qdm12/gluetun/releases/tag/v3.34.0

I can confirm it fixed the connection issues for me.
